content/twim-guide.md

```diff
@@ -27,28 +27,28 @@ Matrix room.
 After joining you should start your post with a mention of the user "TWIM".
 Easiest to do this is to use your client's mention completion feature.
 After that mention everything contained in the message will be part of the entry.
-If you want to post more than one project please split each into individual entries.
+If you want to post more than one project please submit individual entries, one for each project.
 
 Images and other media are supported as well. To add them you should react with
-`⭕` on your media file. Please make sure to do this after your entry so it gets added
-to the correct entry and not on another project of yours.
+`⭕` on your media file. Please make sure to do this right after submitting your text so it gets added
+to the correct entry and not to another project of yours.
 
 ## How to make it a good entry
 
 Making a good entry can have some challenges. Below are some recommendations
-to help the authors to lower their workload when publishing the post.
+to help the editors to lower their workload when publishing the post.
 
 ### Markdown
 
-The bot expected markdown as part of the plaintext event body. Please use a client which
-does send the plaintext as markdown. While we currently accept other formats as well
+The bot expects markdown as part of the plaintext event body. Please use a client which
+does include the markdown in the plaintext. While we currently accept other formats as well
```

Comment:
The wording "send" was actually used as a stricter form of this. Meaning we expect it to include _only_ markdown in the ideal case. "include" could mean that it contains multiple formats which would still need human intervention.

Reply:
I don't understand "sending the plaintext as markdown". Rather I think what you mean is to send markdown (i.e. text with symbols) in the plaintext `body` of message events. That is in contrast to
- HTML `formatted_body`, because zola needs the md
- plain plaintext in the plaintext `body`, i.e. markdown symbols removed

People will/should/have been all along sending messages (in most cases) by typing in markdown, which their client will convert to HTML and include that in the actual event. I don't think there is a reasonable way around that. What you want here is the behavior of element web, which keeps the entered markdown untouched in the body as a fallback.

I actually think that the spec
> The plain text version of the HTML should be provided in the `body`.

specifies that the `body` would not be the original markdown (which is an element and other clients implementation detail to format messages in the first place) but the HTML with all tags stripped.

Reply:
Well we had the case before that we got orgmode instead since thats what the client uses. With then a codeblock for markdown. And I would like to discourage this :)

Reply:
Exactly, it should "include markdown in the plaintext" rather than whatever else. We might also word this entirely different, e.g. "use a client which sends the markdown formatted text in the "plaintext" `body`". It's a family technical requirement that might be confusing for non-technical submitters though.

Reply:
We agreed upon your "use a client which sends the markdown formatted text in the "plaintext" body" suggestion for this in the office hour. Could you update it accordingly? :)

Reply:
```suggestion
sends the markdown formatted text in the `plaintext` body. While we currently accept other formats as well
```